lib/BloqadeExpr/src/interface.jl

```diff
@@ -5,6 +5,16 @@ const n = YaoBlocks.ConstGate.P1
 
 end
 
+function process_atom_positions(atom_positions)
```

Comment:
Would be nice to have a unit test for this (just to keep coverage good, even though it's a very simple function)

Reply:
👍